config/core/deployments/activator.yaml

```diff
@@ -104,13 +104,15 @@ spec:
             httpHeaders:
             - name: k-kubelet-probe
               value: "activator"
-          failureThreshold: 12
+          periodSeconds: 5
+          failureThreshold: 1
         livenessProbe:
           httpGet:
             port: 8012
             httpHeaders:
             - name: k-kubelet-probe
               value: "activator"
+          periodSeconds: 10
```

Comment:
Just curious, why the need for the livenessProbe update?

Reply:
The default is `10` so I just made it explicit.

Cause when I saw the `failureThreshold` of `12` it wasn't obvious (without looking it up) that it would take 120s